deploy/cloudformation/config.go

```diff
@@ -34,10 +34,15 @@ type config struct {
 	EnrollmentToken       string     `mapstructure:"ENROLLMENT_TOKEN"`
 	ElasticArtifactServer *string    `mapstructure:"ELASTIC_ARTIFACT_SERVER"`
 	ElasticAgentVersion   string     `mapstructure:"ELASTIC_AGENT_VERSION"`
-	IntegrationType       *string    `mapstructure:"INTEGRATION"`
 	Dev                   *devConfig `mapstructure:"DEV"`
+	DeploymentType        string     `mapstructure:"DEPLOYMENT_TYPE"`
```

Comment:
It will do it by default.
```golang
func getTemplatePath(deploymentType string, env string) (string, error) {
	if deploymentType == "" {
		// Default is CNVM
		deploymentType = DeploymentTypeCNVM
```